lib/expeditor/command.rb

```diff
@@ -43,7 +43,7 @@ def started?
     def get
       raise NotStartedError if not started?
       @normal_future.get_or_else do
-        if @fallback_var
+        if @fallback_var && @service.fallback_enabled?
```

Comment:
I found another point it affects fallback. Fixed it in 11e55be636907e537e9ab4769a55d115a9e42b89.